### PR TITLE
Added clarifications and corrections

### PR DESCRIPTION
**Pull request by AliceProxy**
## Description
Updated the Prometheus / Grafana documentation to fix a few small issues as well as providing additional detail on a few steps in the guide.

## Changes
Changed AMBASSADOR_HOST to AMBASSADOR_IP to align with its usage in the configuration.
Provided details on obtaining the AMBASSADOR_IP in case of confusion about what IP we are referring to.
Removed the namespace from the port-forward command as we never created that namespace.
Add reminder to change the {{ GRAFANA_NAMESPACE }} as we did with the Ambassador IP 
Update references to the provided Grafana dashboard to the most recent version
Provide steps to add a data source to Grafana which is a required first step before deploying the dashboard

### CODE REVIEW

File: docs/howtos/prometheus.md [hunk 1]
```diff
@@ -384,10 +392,22 @@ spec:
...
-the dashboard ID(10434).
+Before you can import the Ambassador dashboard. You need to add a data source.
+From the Grafana home page, select `Create your first data source`. Now,
+select 'Prometheus'. In the URL section, type in `http://prometheus.default:9090`.
+We deployed prometheus to the default namespace in our example, but if you 
+deployed it to a different namespace, make sure to replace `default` with your
+namespace. Press `Save & Test` to confirm that the data source works.
+
+Import the [provided dashboard](https://grafana.com/grafana/dashboards/4698)
+by clicking the plus sign in the left side-bar, clicking `Import` in the top left, and entering the dashboard ID(4698).
```
Comment (kflynn) [inline docs/howtos/prometheus.md:403]: Why did the dashboard ID change? do we have to worry about it changing again?

**Review by kflynn — COMMENTED** [review]
I have one question, but overall this looks really good, thanks!

File: docs/howtos/prometheus.md (hunk 1 above)
Comment (AliceProxy) [reply docs/howtos/prometheus.md:403]: Not entirely sure. The first time it is mentioned, it links to the updated dashboard which is owned/uploaded by Alex, the second time it is mentioned (and the ID 10434 given) refers to one uploaded by Noah which currently has no overview/description.

Comment (kflynn) [reply docs/howtos/prometheus.md:403]: Yeah, I agree that only one makes sense -- and if we have to take explicit action to make it change, that works out for me. Cool.

**Review by kflynn — DISMISSED** [review]
Hit it! and thanks. 🙂

**Comment by AliceProxy** [comment]
Noah created a new dashboard that is better suited to displaying info from the metrics endpoint. Moved the link to dashboard ID: 4698 to the additional install options (it supports statsd metrics, so it fits in with that content). I also updated the :8877/metrics endpoint page to link to this new dashboard instead of raw json (new dashboard is based off that json).

**Comment by nbkrause** [comment]
> Noah created a new dashboard that is better suited to displaying info from the metrics endpoint. Moved the link to dashboard ID: 4698 to the additional install options (it supports statsd metrics, so it fits in with that content). I also updated the :8877/metrics endpoint page to link to this new dashboard instead of raw json (new dashboard is based off that json).

Didn't create the dashboard but I did add it to the Grafana cloud to make it easier to reference in the docs. The credit for creating it goes to @alexgervais 🙂

File: docs/topics/running/statistics/8877-metrics.md [hunk 2]
```diff
@@ -57,11 +57,10 @@ with Prometheus and Grafana](../../../../howtos/prometheus).
 
 #### Sample dashboard
 
-We provide a [sample Grafana dashboard][`8877-metrics-grafana.json`]
+We provide a [sample Grafana dashboard](https://grafana.com/dashboards/10434)
```
Comment (kflynn) [inline docs/topics/running/statistics/8877-metrics.md:60]: Wait, where does this `10434` come from?

Comment (AliceProxy) [reply docs/topics/running/statistics/8877-metrics.md]: Good catch, thanks. Started to forget which ID I was using with 3 of them in my head now. New standard should be: default to using ID: 13758, keep one link to ID: 4698 at "additional install options" and removing all references to ID: 10434.

**Review by kflynn — APPROVED** [review]
Hit it! 🙂

**Pull request merged**